code/AssetLib/C4D/C4DImporter.cpp

```diff
@@ -305,7 +301,7 @@ void C4DImporter::RecurseHierarchy(BaseObject* object, aiNode* parent) {
 
     // based on Cineware sample code
     while (object) {
-        const LONG type = object->GetType();
```

Comment:
Does this work for32 bit and 64 bit?

Reply:
Yeah, the variable type has been updated to `Int32`, matching the return type of the Cineware SDK, which consistently utilizes a 32-bit length across all platforms. (It's worth noting that while the portability to 32-bit platforms is accounted for, Cineware binaries are only available for 64-bit platforms.)